prometheus-operator/templates/rbac.yaml

```diff
@@ -80,3 +80,27 @@ subjects:
 - kind: ServiceAccount
   name: {{ template "name" . }}
   namespace: {{ .Release.Namespace }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+metadata:
+  name: cluster-admin
```

Comment:
Won't this fail for @rasheedamir as they already have the role `cluster-admin`? @kahootali

Reply:
won't it skip if its there already? it has to be there as we will be deploying to different clusters

Reply:
Most probably, it will give an error that a service account with this name already exists.

Reply:
lets give name like po-cluster-admin

prometheus operator cluster admin